pkg/target/host.go

```diff
@@ -27,20 +27,34 @@ type Host struct {
 	Wm   sync.Mutex
 	stop chan int
 
-	Lg                      *zap.Logger
-	Ms                      *metrics.Prom
-	SendTimeoutSec          uint32
-	ConnTimeoutSec          uint32
-	TCPOutBufFlushPeriodSec uint32
+	updateHostHealthStatus chan *HostStatus
+
+	Lg                        *zap.Logger
+	Ms                        *metrics.Prom
+	SendTimeoutSec            uint32
+	ConnTimeoutSec            uint32
+	KeepAliveSec              uint32
+	MaxReconnectPeriodMs      uint32
+	ReconnectPeriodDeltaMs    uint32
+	ConnectionLossThresholdMs uint32
+	TCPOutBufFlushPeriodSec   uint32
```

Comment:
it makes sense to group these. maybe, consider passing a pointer to the full config

Reply:
The main config includes config for target and listening port, which the host doesn't need to know.

Reply:
Apart from other configs.

Reply:
we can do it later